scripts/android_package.sh

```diff
@@ -151,7 +151,6 @@ if [[ "$RELEASE" ]]; then
     CONFIG-=debug_and_release \
     CONFIG+=release \
     $PRODMODE \
-    ANDROID_ABIS="armeabi-v7a x86 arm64-v8a" \
```

Comment:
While i was at it, we have those defined in the qt project file, no need to have them here.